src/SonarScanner.MSBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -39,6 +39,7 @@ public static class AnalysisConfigGenerator
             IDictionary<string, string> serverProperties,
             List<AnalyzerSettings> analyzersSettings,
             ISonarQubeServer sonarQubeServer,
+            string unchangedFilesPath,
             ILogger logger)
```

Comment:
The logger is not used and can be removed.

Reply:
LOL Nice catch